pkg/cri/resource-manager/policy/builtin/memtier/syscall_linux.go

```diff
@@ -56,8 +56,8 @@ func movePagesSyscall(pid int, count uint, pages []uintptr, nodes []int, flags i
 		err = unix.Errno(en)
 	}
 
-	log.Debug("move_pages(): pid %d, count %d, pages %v, nodes %v, flags %d: return value %d, status %d, errno %v",
-		pid, count, pages, nodes, flags, uint(ret), cStatus, err)
+	// log.Debug("move_pages(): pid %d, count %d, pages %v, nodes %v, flags %d: return value %d, status %d, errno %v",
+	// 	pid, count, pages, nodes, flags, uint(ret), cStatus, err)
```

Comment:
You could be right. On the other hand, probably everyone would want to have `memtier` debug messages every time they were using `demoter` messages.

Maybe one possibility would be to have a new log level: `log.Periodic()`? Then the user could choose the log level so that periodic messages would not be shown.

Reply:
Hmm. I would then rather
  1. separate the periodic scanning to a logger source of its own (demoter-scan)
  2. add logger support for one source to imply/suggest another so that turning on debugging would turn on debugging for all those implied/suggested sources, which are not explicitly disabled (and logger source implications would be both programmatic and possible to override with configuration).
  3. make demoter imply memtier and scanning

This would make it possible to achieve all possible combinations of memtier/demoter/scanning without introducing new log levels, which are golang logger antipattern, and have intuitive default grouping while providing the necessary fine-grained control to override the defaults.

Reply:
We can reconsider this when we swap in the new klog-based logger.